restlicsharpclient/restliclientintegrationtest/RestClientIntegrationTests.cs

```diff
@@ -16,28 +16,34 @@
 
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
+using System;
+
 using com.linkedin.restli.test.api;
 using restlicsharpclient.restliclient;
 using restlicsharpclient.restliclient.request;
 using restlicsharpclient.restliclient.request.builder;
 using restlicsharpclient.restliclient.response;
-using restlicsharpclient.restliclient.response.decoder;
 
 namespace restlicsharpclient.restliclientintegrationtest
 {
     [TestClass]
-    public class RestClientTests
+    public class RestClientIntegrationTests
     {
         [TestMethod]
-        public void RestClient_GetGreeting()
+        public void RestClientIntegration_GetGreeting()
```

Comment:
There is no particular C# test naming convention, though I do this because it makes it easy when Visual Studio lists all the tests in the side panel UI. I had to look up how to sort test results by class. There is a way to do it, so I will remove these prefixes.